templates/CRM/Pledge/Form/Pledge.tpl

```diff
@@ -70,7 +70,12 @@
   </tr>
         <tr class="crm-pledge-form-block-installments">
       <td class="label">{$form.installments.label}</td>
-      <td>{$form.installments.html} {ts}installments of{/ts} {if $action eq 1 or $isPending}{$form.eachPaymentAmount.html|crmMoney:$currency}{elseif $action eq 2 and !$isPending}{$eachPaymentAmount|crmMoney:$currency}{/if}&nbsp;{ts}every{/ts}&nbsp;{$form.frequency_interval.html}&nbsp;{$form.frequency_unit.html}</td></tr>
+      <td>{$form.installments.html} {ts}installments of{/ts}
+        {if $action eq 1 or $isPending}
+         <span class="currency-symbol">{$form.eachPaymentAmount.html|crmMoney:$currency}</span>
+        {elseif $action eq 2 and !$isPending}
+          <span class="currency-symbol">{$eachPaymentAmount|crmMoney:$currency} </span>
+        {/if}&nbsp;{ts}every{/ts}&nbsp;{$form.frequency_interval.html}&nbsp;{$form.frequency_unit.html}</td></tr>
```

Comment:
No, I added span with class `<span class="currency-symbol">`, so that I can change it when currency drop down changes.